source/iostreams.tex

```diff
@@ -13592,11 +13592,8 @@
 \pnum
 \defnx{Pathname resolution}{pathname resolution}
 is the operating system dependent mechanism for resolving
-a pathname to a particular file in a file hierarchy. There may be multiple
-pathnames that resolve to the same file.
-\begin{example}
-POSIX specifies the mechanism in section 4.12, Pathname resolution.
-\end{example}
+a pathname to a particular file in a file hierarchy (see POSIX, section 4.12).
```

Comment:
This seems to suggest that POSIX defines the mechanism, but that's only true for POSIX-based operating systems. i.e., this seems to contradict the first half of the sentence.

```suggestion
a pathname to a particular file in a file hierarchy.
\begin{example}
For POSIX-based operating systems, this mechanism is specified in POSIX section 4.12).
\end{example}
```

I'm a little sad to lose the "Pathname Resolution" part, because while "POSIX 4.12" unambiguously means "ISO/IEC/IEEE 9945:2009 4.12" in this document, when POSIX 2024 gets published in the near future the correct section will be different (it's 4.16 in the current 202x draft 4 doc). If we also name the section then somebody consulting the latest POSIX spec will be able to find it more easily, without having to check what 4.12 in older spec corresponds to. I realise that checking the older spec is more correct, since that's our normative reference. But people often want to consult the latest spec, since that might be what their implementation's libc actually conforms to.

Reply:
Yeah, this is less of a change. Applied.